### PR TITLE
6158: Basic index page

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/includes/layout.njk [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="ie8" lang="en"><![endif]-->
+<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
+<!--[if gt IE 9]><!--><html lang="en"><!--<![endif]-->
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="{% if pageDescription %}{{ pageDescription }}{% else %}NHSD Buying Catalogue Admin{% endif %}">
```
Comment (kat1906) [inline app/includes/layout.njk:9]: rename

File: app/includes/layout.njk [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    <title>{% if pageTitle %}{{ pageTitle }} - {% endif %}NHSD Buying Catalogue Admin</title>
```
Comment (kat1906) [inline app/includes/layout.njk:11]: rename

File: app/pages/index/controller.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { getContext } from './contextCreator';
+
+export const getIndexContext = async () => getContext();
```
Comment (kat1906) [inline app/pages/index/controller.js:3]: don't need async

File: app/includes/layout.njk (hunk 1 above)
Comment (BenBJSS) [reply app/includes/layout.njk:9]: done

File: app/includes/layout.njk (hunk 2 above)
Comment (BenBJSS) [reply app/includes/layout.njk:11]: done

File: app/test-utils/testHarness.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (alibombaye) [inline app/test-utils/testHarness.js:1]: what's this one about? do we still need it

File: app/pages/index/controller.js (hunk 3 above)
Comment (BenBJSS) [reply app/pages/index/controller.js:3]: removed

File: app/test-utils/testHarness.js (hunk 4 above)
Comment (kat1906) [reply app/test-utils/testHarness.js:1]: needed it for browsersync if I remember correctly so can get binned if that is the case

Comment (alibombaye) [reply app/test-utils/testHarness.js:1]: :) yes let's tidy that up.

Comment (BenBJSS) [reply app/test-utils/testHarness.js:1]: removed

**Pull request merged**